projects/python_packages.cmake

```diff
@@ -48,7 +48,9 @@ else()
             COMMAND ${Python3_EXECUTABLE} -m pip install http://software.ultimaker.com/cura-binary-dependencies/numpy-1.15.4+mkl-cp35-cp35m-win32.whl
             COMMAND ${Python3_EXECUTABLE} -m pip install http://software.ultimaker.com/cura-binary-dependencies/scipy-1.2.0-cp35-cp35m-win32.whl
             COMMAND ${Python3_EXECUTABLE} -m pip install http://software.ultimaker.com/cura-binary-dependencies/Shapely-1.6.4.post1-cp35-cp35m-win32.whl
-            COMMENT "Install Numpy, Scipy, and Shapely"
+            # pywin32 is required to provide the keyring library with access to the Windows Credential Manager
```

Comment:
How would this even work when it's only added to the Win32 build, not the amd64 one as well?

Reply:
You are absolutely right. I placed it in the wrong place